web/views.py

```diff
@@ -68,10 +63,10 @@ def compare(request):
 
             lang2_friendly_name = lang2_file_json["meta"]["language_name"]
             lang2_categories = lang2_file_json["categories"]
-            lang2_concepts = lang2_file_json[concept]
+            lang2_concepts = lang2_file_json[concept_query_string]
 
     except:
-        return HttpResponse("The " + request.GET.get('concept', '') + " concept of either the " + request.GET.get('lang1', '') + " or " + request.GET.get('lang2', '') + " languages doesn't exist or hasn't been implemented yet.")
+        return HttpResponseBadRequest("The " + concept_query_string + " concept of either the " + lang1_query_string + " or " + lang2_query_string + " languages doesn't exist or hasn't been implemented yet.")
```

Comment:
Same argument for having this be a 404 instead of a 400

Reply:
Also will change!